internal/store/build_result.go

```diff
@@ -442,7 +442,7 @@ func RunningContainersForTargetForOnePod(iTarget model.ImageTarget, runtimeState
 		if c.ImageRef == nil || iTarget.Refs.ClusterRef().Name() != c.ImageRef.Name() {
 			continue
 		}
-		if c.ID == "" || c.Name == "" || !c.Ready {
+		if c.ID == "" || c.Name == "" {
```

Comment:
hmmm... I think we want to check if the container is Running, not Ready?
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-states

The reason this check was added is because there are a variety of cases where the container won't be live-updatable (e.g., the container crashed, or the image isn't pulling correctly)